oak-run/src/main/java/org/apache/jackrabbit/oak/run/RevisionsCommand.java

```diff
@@ -320,15 +330,13 @@ private VersionGarbageCollector bootstrapVGC(RevisionsOptions options,
         return gc;
     }
 
-    private void info(RevisionsOptions options, Closer closer)
-            throws IOException {
+    private void info(RevisionsOptions options, Closer closer) throws IOException {
```

Comment:
Similar here - these are formatting changes that increase change surface making porting effort more expensive.